tsconfig.json

```diff
@@ -1,38 +1,27 @@
 {
+  "extends": "./tsconfig.base.json",
   "compilerOptions": {
-    "jsx": "react",
-    "target": "esnext",
-    "lib": ["dom", "dom.iterable", "esnext"],
-    "allowSyntheticDefaultImports": true,
-    "module": "commonjs",
-    "isolatedModules": true,
-    "strict": true,
-    "moduleResolution": "node",
-    "skipLibCheck": true,
     "paths": {
       "*": [
         "./frontend/src/*",
         "./frontend/test/*",
         "./enterprise/frontend/src/*",
         "./enterprise/frontend/test/*"
-      ],
-      "e2e/*": ["./e2e/*"]
-    },
-    "allowJs": true,
-    "esModuleInterop": true,
-    "experimentalDecorators": true,
-    "forceConsistentCasingInFileNames": true,
-    "resolveJsonModule": true
+      ]
+    }
   },
   "include": [
-    "frontend/src/**/*.ts",
-    "frontend/src/**/*.tsx",
-    "enterprise/frontend/src/**/*.ts",
-    "enterprise/frontend/src/**/*.tsx",
-    "frontend/test/**/*.ts",
-    "frontend/test/**/*.tsx",
-    "enterprise/frontend/test/**/*.ts",
-    "enterprise/frontend/test/**/*.tsx"
+    "frontend/src/**/*",
```

Comment:
This is the gist of this PR, loads all the JS files into TypeScript, so intellisense works.